source/macos/AdaptiveCards/AdaptiveCards/Utils/ACOVisibilityContext.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ACOVisibilityContext.swift
+//  AdaptiveCards
+//
+//  Created by uchauhan on 05/09/22.
+//
+// This Visibility Context is used to store and retrieve specific protocol-adopted content views. This context will be preserved within a single card.
+
+import AdaptiveCards_bridge
+import AppKit
+
+class ACOVisibilityContext: NSObject {
+    /// store view with its corresponding root content view manager
+    /// `Key` : View Id Store with Strong referance
+    /// `Value`: corresponding root content view manager Store with Weak referance
+    private let visibilityMap: NSMapTable<NSString, ACSIVisibilityManagerFacade>
+    
+    override init() {
+        visibilityMap = NSMapTable<NSString, ACSIVisibilityManagerFacade>(keyOptions: .strongMemory, valueOptions: .weakMemory)
```

Comment:
If we have .strongMemory here, do we need an explicit deinit?

Reply:
strong memory reference is NSString, so we do not need to deinit explicitly.